apps/researcher/src/app/[locale]/(objects)/objects/[id]/page.tsx

```diff
@@ -112,171 +112,178 @@ export default async function Details({params}: Props) {
-              enrichmentType={AdditionalType.Type}
-            >
-              {object.types?.map(type => <div key={type.id}>{type.name}</div>)}
-            </Metadata>
+              <Metadata
+                translationKey="types"
+                enrichmentType={AdditionalType.Type}
+              >
+                {object.types?.map(type => (
+                  <div key={type.id}>{type.name}</div>
+                ))}
+              </Metadata>
 
-            <Metadata
-              translationKey="techniques"
-              enrichmentType={AdditionalType.Technique}
-            >
-              {object.techniques?.map(technique => (
-                <div key={technique.id}>{technique.name}</div>
-              ))}
-            </Metadata>
+              <Metadata
+                translationKey="techniques"
+                enrichmentType={AdditionalType.Technique}
+              >
+                {object.techniques?.map(technique => (
+                  <div key={technique.id}>{technique.name}</div>
+                ))}
+              </Metadata>
 
-            <Metadata
-              translationKey="creators"
-              enrichmentType={AdditionalType.Creator}
-            >
-              {object.creators?.map(creator => (
-                <div key={creator.id}>{creator.name}</div>
-              ))}
-            </Metadata>
+              <Metadata
+                translationKey="creators"
+                enrichmentType={AdditionalType.Creator}
+              >
+                {object.creators?.map(creator => (
+                  <div key={creator.id}>{creator.name}</div>
+                ))}
+              </Metadata>
 
-            <Metadata
-              translationKey="inscriptions"
-              enrichmentType={AdditionalType.Inscription}
-            >
-              {object.inscriptions?.map(inscription => (
-                <div key={inscription}>{inscription}</div>
-              ))}
-            </Metadata>
-          </div>
-        </main>
-        <aside className="w-full md:w-1/3 self-stretch order-1 md:order-2 md:mx-0  p-1 flex flex-col justify-start md:bg-blueGrey-50">
-          {galleryImages.length > 0 && (
-            <Gallery
-              organizationName={organization?.name}
-              images={galleryImages}
-            />
-          )}
-        </aside>
+              <Metadata
+                translationKey="inscriptions"
+                enrichmentType={AdditionalType.Inscription}
+              >
+                {object.inscriptions?.map(inscription => (
+                  <div key={inscription}>{inscription}</div>
+                ))}
+              </Metadata>
+            </div>
+          </main>
+          <aside className="w-full md:w-1/3 self-stretch order-1 md:order-2  md:mx-0 md:bg-consortiumBlue-900 p-1">
+            {galleryImages.length > 0 && (
+              <div className="flex flex-row md:flex-col gap-1 sticky top-4">
+                <Gallery images={galleryImages} />
```

Comment:
The license of images has been added recently. The grey tile colour is not readable on the blue background.